docs/Field-Mapping.md

```diff
@@ -283,22 +283,30 @@ All payment methods:
     * `clearing_bankbic` ->  `paidFromBIC` 
    
 `INVOICE-KLARNA`:
-  * _Required_ `language` -> custom field `languageTag` of Type String on the CT Payment
-    * (may be moved to the Order, Cart and Customer Objects later)
+  * _Required_ 
+    * `language` -> custom field `languageTag` of Type String on the CT Payment
```

Comment:
Link for implementation detail is ok but i think we should document also here in which order language is being deduced and which ISO standard is expected.